packages/composables/src/hooks/useUser.ts

```diff
@@ -109,7 +124,55 @@ export const useUser = (): UseUser => {
     return getCustomerOrderDetails(orderId);
   };
 
-  const getAddresses = (): Promise<CustomerAddress[]> => getCustomerAddresses();
+  const loadAddresses = async (): Promise<void> => {
+    try {
+      addresses.value = await getCustomerAddresses();
+    } catch (e) {
+      error.value = e.message;
+    }
+  };
+
+  const markAddressAsDefault = async ({
+    addressId,
+    type
+  }: {
+    addressId?: string;
+    type?: AddressType;
+  }): Promise<boolean> => {
+    if (!addressId || !type) {
+      return false;
+    }
+
+    try {
+      switch (type) {
```

Comment:
```suggestion
     if (type === AddressType.billing) {   // or even "billing"
       await setDefaultCustomerBillingAddress(addressId);
     } else {
       await setDefaultCustomerShippingAddress(addressId);
     }
     await refreshUser();
     return true;
```

Reply:
I made a little refactor down here :+1:  I've left a switch, in this case, is more readable. 
I rather use if -else for boolean values, for data, I would stay with the switch, which is also even faster than ifs ;-)